django/forms/widgets.py

```diff
@@ -9,7 +9,9 @@
 import re
 from itertools import chain
 
+from django.apps import apps
 from django.conf import settings
+from django.contrib.staticfiles.storage import staticfiles_storage
```

Comment:
In general django doesn't allow contrib imports from core, but it might be worth making an exception in this case.